test/distributed_exec.jl

```diff
@@ -48,7 +48,7 @@ end
 
 # Test that the client port is reused. SO_REUSEPORT may not be supported on
 # all UNIX platforms, Linux kernels prior to 3.9 and older versions of OSX
-if is_unix()
+if ccall(:jl_has_so_reuseport, Int32, ()) == 1
```

Comment:
The lines below were checking if reuseport is supported. Can be simplified now to directly call `reuseport_tests()` if supported else print an informational string.